redbot/core/commands/help.py

```diff
@@ -289,13 +290,32 @@ def group_embed_fields(fields: List[EmbedField], max_chars=1000):
         pages = []
 
         page_char_limit = await ctx.bot._config.help.page_char_limit()
+        page_char_limit = min(page_char_limit, 5990)  # Just in case someone was manually...
+
+        author_info = {"name": f"{ctx.me.display_name} Help Menu", "icon_url": ctx.me.avatar_url}
+
+        # Offset calculation here is for total embed size limit
+        # 20 accounts for# *Page {i} of {page_count}*
+        offset = len(author_info["name"]) + 20
+        foot_text = embed_dict["footer"]["text"]
+        if foot_text:
+            offset += len(foot_text)
+        offset += len(embed_dict["description"])
```

Comment:
This throws an error any way I try to run `[p]help`

```py
[2019-12-26 14:30:56] [ERROR] red: Exception in command 'help'
Traceback (most recent call last):
  File "<venv>\lib\site-packages\redbot\core\commands\help.py", line 644, in red_help
    await ctx.bot.send_help_for(ctx, thing_to_get_help_for)
  File "<venv>\lib\site-packages\redbot\core\bot.py", line 436, in send_help_for
    return await self._help_formatter.send_help(ctx, help_for)
  File "<venv>\lib\site-packages\redbot\core\commands\help.py", line 143, in send_help
    await self.format_command_help(ctx, help_for)
  File "<venv>\lib\site-packages\redbot\core\commands\help.py", line 233, in format_command_help
    await self.make_and_send_embeds(ctx, emb)
  File "<venv>\lib\site-packages\redbot\core\commands\help.py", line 303, in make_and_send_embeds
    offset += len(embed_dict["description"])
KeyError: 'description'
```

Reply:
*sigh* of course. Give me a sec, I know why that's happening.